sdk/servicebus/service-bus/test/README.md

```diff
@@ -120,14 +120,12 @@ AAD_CLIENT_SECRET=""
 AAD_TENANT_ID=""
 RESOURCE_GROUP=""
 AZURE_SUBSCRIPTION_ID=""
-CLEAN_NAMESPACE="true"
 ```
 
 **Note:**
 * `RESOURCE_GROUP` and `AZURE_SUBSCRIPTION_ID` can be found at your servicebus-namespace in the azure-portal.
```

Comment:
`RESOURCE_GROUP` and `AZURE_SUBSCRIPTION_ID` are not needed by the tests anymore after the changes in #6390

Reply:
Updated